src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: auto;
```

i'm not sure if the behavior of an element with the value `auto` can be understood by other developers, let's set the fixed `grid-template-columns` for our grid
```suggestion
  grid-template-columns: $card-width;
```

---

yes, you are right, it will be better to add variable
thanks
fixed
Have a nice weekend)